examples/basic/app/(app)/page.tsx

```diff
@@ -1,3 +1,9 @@
+'use client'
+//Once this page starts to get built, make this can be server component again
 export default function Index() {
+  if (typeof window !== 'undefined') {
```

Comment:
This wouldn't work in local development, so maybe the better approach is to use [`useRouter`](https://nextjs.org/docs/app/api-reference/functions/use-router) and a `useEffect` call to perform a client-side redirect to `/swingset`?